### PR TITLE
[ZT] add DLP for google drive

### PR DESCRIPTION
**Pull request by ranbel**
PCX-4472

### CODE REVIEW

**Comment by ranbel** [comment]
Preview link: https://597f6283.cloudflare-docs-7ou.pages.dev/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/configuration-guides/

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/configuration-guides/google-drive.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+title: Google Drive
+weight: 1
+layout: single
+---
+
+# DLP for Google Drive
+
+Cloudflare Data Loss Prevention can inspect upload and download traffic to and from Google Drive.
+
+## Policy recommendations
```
Comment (cloudflare-noelle) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/configuration-guides/google-drive.md]: I know I took way too long to review this, but with the wait, we can now remove lines 12-22.  We removed these URLs from scanning in a quick SHIP to get ready for GA.

**Review by cloudflare-noelle — CHANGES_REQUESTED** [review]
One change recommended based on quick ship, thanks sooo much for your patience here

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ weight: 5
 layout: single
 ---
 
-# Data Loss Prevention (beta)
+{{<beta>}} Data Loss Prevention {{</beta>}}
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md:8]: @cloudflare-noelle Should we remove the beta flag?

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/configuration-guides/google-drive.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Cloudflare Data Loss Prevention can inspect upload and download traffic to and from Google Drive.
+
+## Configuring for compatibility
+
+### Browsers
+
+DLP for Google Drive is compatible with all popular browsers.
+
+To inspect Google Drive traffic in Chrome, you will need to [disable QUIC in Chrome](/cloudflare-one/policies/filtering/http-policies/incompatible-traffic/#disable-quic-in-google-chrome).
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/configuration-guides/google-drive.md]: Do users still need to disable QUIC, now that Gateway automatically configures Chrome to fall back to H2?
https://developers.cloudflare.com/cloudflare-one/policies/filtering/http-policies/http3/

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/configuration-guides/google-drive.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+# DLP for Google Drive
+
+Cloudflare Data Loss Prevention can inspect upload and download traffic to and from Google Drive.
+
+## Configuring for compatibility
+
+### Browsers
+
+DLP for Google Drive is compatible with all popular browsers.
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/configuration-guides/google-drive.md]: Which browsers are supported?

Comment (cloudflare-noelle) [reply content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/configuration-guides/google-drive.md]: The site or application being used is more likely to affect visibility than the browser. I have tested on Chrome, Firefox, and Safari, and I haven't received any feedback from customers on issues with a different browser.

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/configuration-guides/google-drive.md (hunk 3 above)
Comment (cloudflare-noelle) [reply content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/configuration-guides/google-drive.md]: Oo good catch here, we can pull this line too, thanks!

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md (hunk 2 above)
Comment (cloudflare-noelle) [reply content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md:8]: DLP is going GA on Monday the 19th, would it be possible to pull the beta flag next friday or that Monday?

Comment (ranbel) [reply content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md:8]: yes let's pull it on Monday the 19th (I'm OOO on Friday)

**Review by kodster28 — APPROVED** [review]

**Pull request merged**